### PR TITLE
Portfolio : Added form 

### PR DESCRIPTION
**Pull request by YannickZahinda**
>>Portfolio: add a contact form and finish the mobile version

1) Added a form in the footer to complete the mobile version
2) Added the form validation with HTML 5 forms validations
3) Sticked to the figma design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -249,25 +249,29 @@ <h4 class="contact-us-heading">CONTACT US</h4>
             feature that you need built or a project that needs coding. I’d love
             to help with it
           </p>
-          <form id="my-form">
-            <input type="text" placeholder="Text input" id="text-input" />
+          <form method="post" id="my-form">
```
Comment (gemmen29) [inline index.html:252]: Please add the `action` attribute and make it similar to `https://formspree.io/f/{form_id}`.  Kindly use the site provided in the requirement of the project.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @YannickZahinda, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -249,25 +249,29 @@ <h4 class="contact-us-heading">CONTACT US</h4>
             feature that you need built or a project that needs coding. I’d love
             to help with it
           </p>
-          <form id="my-form">
-            <input type="text" placeholder="Text input" id="text-input" />
+          <form action="https://formspree.io/f/form_id" method="post" id="my-form">
```
Comment (Deepakdanger) [inline index.html:252]: - [ ] In `Form` `action`, You have to create an ID on formspree.io and put it in place of `form_id`.Then your form is submitted successfully. Now it's showing `Form Not Found` after submitting. Let's FIx this.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @YannickZahinda, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**